shared/src/main/java/org/pac4j/lagom/Authorizers.java

```diff
@@ -0,0 +1,95 @@
+package org.pac4j.lagom;
+
+import org.pac4j.core.authorization.authorizer.Authorizer;
+import org.pac4j.core.authorization.authorizer.IsAnonymousAuthorizer;
+import org.pac4j.core.authorization.authorizer.IsAuthenticatedAuthorizer;
+import org.pac4j.core.authorization.authorizer.RequireAnyRoleAuthorizer;
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.profile.CommonProfile;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Fluent API for creating authorizers in implementation Lagom services.
+ *
+ * @author Sergey Morgunov
+ * @since 1.0.0
+ */
+public final class Authorizers {
+
+    public static <U extends CommonProfile> Authorizer<U> authenticated() {
+        return new IsAuthenticatedAuthorizer<>();
+    }
+
+    public static <U extends CommonProfile> Authorizer<U> anonymous() {
+        return new IsAnonymousAuthorizer<>();
+    }
+
+    public static <U extends CommonProfile> Authorizer<U> anyRole(String ... roles) {
+        return new RequireAnyRoleAuthorizer<>(roles);
+    }
+
+    public static <U extends CommonProfile> Authorizer<U> anyRole(List<String> roles) {
+        return new RequireAnyRoleAuthorizer<>(roles);
+    }
+
+    public static <U extends CommonProfile> Authorizer<U> anyRole(Set<String> roles) {
```

Comment:
`requireAnyRole`. I don't see the `requireAllRoles` even if I guess that the `and` handles this use case...

Reply:
Done